runtime/browser/xwalk_browser_context.h

```diff
@@ -97,6 +97,12 @@ class XWalkBrowserContext
       bool in_memory,
       content::ProtocolHandlerMap* protocol_handlers,
       content::URLRequestInterceptorScopedVector request_interceptors);
+  void InitFormDatabaseService();
+  XWalkFormDatabaseService* GetFormDatabaseService();
+  void CreateUserPrefServiceIfNecessary();
+  void UpdateAcceptLanguages(const std::string& accept_languages);
+  void SetSaveFormData(bool enable) { save_form_data_ = enable; }
+  bool SaveFormDataEnabled() const { return save_form_data_; }
```

Comment:
if (!XWalkBrowserContext::GetDefault()->SaveFormDataEnabled()) in xwalk_content_base. If the user wants to turn it off then we don't provide autofill neither we save the form data.